### PR TITLE
Improve explanations and structure around devnet/testnet

### PR DESCRIPTION
**Pull request by ma2bd**

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -43,13 +43,13 @@
...
 
-- [Operators](operators.md)
+- [Node Operators](operators.md)
 
   - [Devnets](operators/devnets.md)
     - [Docker Compose](operators/devnets/compose.md)
     - [kind](operators/devnets/kind.md)
-  - [Validators](operators/validators.md)
-    - [Joining an existing Testnet](operators/validators/joining.md)
+  - [Testnets](operators/validators.md)
```
Comment (afck) [inline src/SUMMARY.md:51]: Maybe the file should also be renamed to `testnets.md`.

File: src/operators/devnets.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
 # Devnets
 
-This section discusses how to deploy developer networks, aka "Devnets".
+This section discusses how to deploy developer networks, aka "Devnets", for
+testing and development purposes.
+
+Devnets always starts from a genesis configuration and an empty state. Validator
```
Comment (afck) [inline src/operators/devnets.md:6]:
```suggestion
Devnets always start from a genesis configuration and an empty state. Validator
```

**Review by afck — APPROVED** [review]

File: src/operators/validators.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
...
+# Testnets
 
-This section discusses how to join an existing Testnet by deploying a validator.
+This section discusses how to deploy a validator node and join an existing
+Testnet.
 
-> Note: The initial Testnets are invite-only managed by the Linera Protocol core
-> team.
+In a Testnet, the validator nodes are run by different operators. Testnets are
+expected to gain in stability and decentralization over time in preparation of
```
Comment (christos-h) [inline src/operators/validators.md:7]: Testnet stability and decentralization will improve over time...

**Review by christos-h — APPROVED** [review]
Nice!

**Pull request merged**